fast64_internal/f3d/f3d_writer.py

```diff
@@ -1496,7 +1496,7 @@ def saveOrGetF3DMaterial(material, fModel, obj, drawLayer, convertTextureData):
         if f3dMat.tex1.tex is None and not f3dMat.tex1.use_tex_reference:
             raise PluginError('In material "' + material.name + '", a texture has not been set.')
 
-        fMaterial.useLargeTextures = not loadTextures
+        fMaterial.useLargeTextures = useLargeTextures
```

Comment:
I think `useLargeTextures` may be unbound here, for example if texture 1 is set but texture 0 isn't